bessctl/conf/perftest/pktgen_http.bess

```diff
@@ -50,7 +50,7 @@ for i in range(num_cores):
     #rewrite the last two octets of the source IP.
     src \
     -> rw \
-    -> RandomUpdate(fields=[{'offset': 14, 'size': 2, 'min': 1, 'max': num_flows}]) \
+    -> RandomUpdate(fields=[{'offset': 30, 'size': 2, 'min': 1, 'max': num_flows}]) \
```

Comment:
The offset doesn't seem to match the comment. At offset 30 there is a destination IP address.